src_extra/networklayer_tcp.c

```diff
@@ -35,7 +35,6 @@
 
 /* workaround a glibc bug where an integer conversion is required */
 #if !defined(_WIN32)
-# include <features.h>
```

Comment:
It looks as if features.h is included with stdio.h when glibc is used.
And since stdio.h is used, just remove the line as-is and it should be fine everywhere.